breaking/entrypoint.sh

```diff
@@ -88,3 +85,7 @@ else
 fi
 
 echo "$delimiter" >>"$GITHUB_OUTPUT"
+
+if [ "$fail_on_diff" = "true" -a -n "$breaking_changes" ]; then
+    exit 1
+fi
```

Comment:
@alarvi, thanks for the PR.
I agree that it's better to write the logs first in case of failure. However, the current implementation is handling logic that should be part of oasdiff. What if tomorrow it fails due to different logic? We would then need to implement that in both repositories. Additionally, what if oasdiff incorporates logic for response error codes?

Reply:
Fair point. What would you suggest oasdiff does differently? I guess the ideal case for the action is that it never needs to call oasdiff twice, right?

Beyond that, what alternatives do we have to achieve both having the workflow fail and logs produced? I took inspiration from the tests and added this extra step to our workflow:
```
      - name: Check if there are any breaking changes
        run: |
          delimiter=$(cat /proc/sys/kernel/random/uuid | tr -d '-')
          output=$(cat <<-$delimiter
          ${{ steps.oasdiff.outputs.breaking }}
          $delimiter
          )
          if [ "$output" != "No breaking changes" ]; then
            echo "There are breaking changes in the OpenAPI spec"
            echo "$output"
            exit 1
          else
            echo "No breaking changes in the OpenAPI spec"
          fi
```
But now we have coupled the workflow with the output format of oasdiff, which isn't ideal.

Reply:
@alarvi, as you mentioned, I see two options here:

1. Calling `oasdiff breaking` twice: once to calculate breaking changes and write logs, then check if `fail_on_diff` is `true` and run it again.
2. Running without `fail_on_diff` and using the breaking action output. In this case, it is coupled with the oasdiff **action** output message: "No breaking changes." Is that what you mean? If yes, you are correct, if we change this message, your code will break. We can add a comment to this message and a dedicated test that will fail in case this message changes.

Although it's not a perfect solution, I prefer the first one. What do you think?